acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/DrivingLicenceAPIPage.java

```diff
@@ -81,6 +82,12 @@ public void dlUserIdentityAsJwtString(String criId, Integer rowNumber)
         String coreStubUrl = configurationService.getCoreStubUrl(false);
         SESSION_REQUEST_BODY = createRequest(coreStubUrl, criId, jsonString);
         LOGGER.info("SESSION_REQUEST_BODY = " + SESSION_REQUEST_BODY);
+
+        // Capture client id for using later in the auth request
+        Map<String, String> deserialisedSessionResponse =
+                objectMapper.readValue(SESSION_REQUEST_BODY, new TypeReference<>() {});
+        CLIENT_ID = deserialisedSessionResponse.get("client_id");
+        LOGGER.info("CLIENT_ID = " + CLIENT_ID);
```

Comment:
Sonar will complain about this again that we should be using string formatting

Reply:
Thanks James, updated the Logger Line to reflect this change